src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -332,6 +332,7 @@
     <!-- This is important because otherwise the VSSDK will include the unsigned version coming from the global packages folder instead of the in-place signed one from the bootstrapped packages folder -->
     <PackageReference Include="Lucene.Net" ExcludeAssets="all" />
     <PackageReference Include="Newtonsoft.Json" NoWarn="NU1605" />
+    <PackageReference Include="SharpZipLib" ExcludeAssets="all" /> <!-- dependency of Lucene.net. Can delete when Lucene.net has an update with a newer dependency on sharpziplib -->
```

Comment:
Does NuGet need to ship this dll in VS? 
Is another component bringing it? Is it a high enough version?

Reply:
Interestingly, We don't currently ship `ICSharpCode.SharpZipLib.dll` in our vsix. Also see https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixignore#L4

And my VS install doesn't have that file anywhere. So, I guess that we don't use whatever Lucene.Net functionality needs SharpZipLib.

In any case, I believe that this PR is not changing any build outputs, just keeping component governance happy.

Reply:
So this is for the component governance alert at https://devdiv.visualstudio.com/DevDiv/_componentGovernance/NuGet-NuGet.Client-Trusted?_a=alerts&typeId=5672187&alerts-view-option=active, right?
Is processing the alerts going to be a regular process?

Reply:
> I guess that we don't use whatever Lucene.Net functionality needs SharpZipLib.

Fine by me. Thanks for validating